src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -335,8 +335,23 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 
 <!-- Do not resolve from the GAC under any circumstances in Mobile -->
 <PropertyGroup>
-	<AssemblySearchPaths>$([System.String]::Copy('$(AssemblySearchPaths)').Replace('{GAC}',''))</AssemblySearchPaths>
-	<AssemblySearchPaths Condition="'$(MSBuildRuntimeVersion)' != ''">$(AssemblySearchPaths.Split(';'))</AssemblySearchPaths>
+	<AssemblySearchPaths>
+		{HintPathFromItem};
+		{TargetFrameworkDirectory};
+		{RawFileName};
+	</AssemblySearchPaths>
+	<AllowedReferenceAssemblyFileExtensions>
+		.dll;
+		.exe;
```

Comment:
not sure we need `.exe` for XA. Since none of our apps or projects use it.

Reply:
Yeah, I think we can drop `.exe`. This might improve build times a bit further.